pkg/controllers/node/termination/controller.go

```diff
@@ -132,6 +160,23 @@ func (c *Controller) deleteAllNodeClaims(ctx context.Context, node *v1.Node) err
 	return nil
 }
 
+func (c *Controller) waitForVolumeDetachments(ctx context.Context, node *v1.Node, backoff wait.Backoff) error {
+	err := wait.ExponentialBackoffWithContext(ctx, backoff, func(ctx context.Context) (bool, error) {
```

Comment:
Is there a better polling mechanism to use than wait.ExponentialBackoff? I don't think a reconciler requeue works here because we WANT to give up on waiting for volumeattachment objects to be deleted if it takes too long.

This is because adding the out of service taint is enough to prevent the 6+ delay. (albeit not the 1+ minute delay)

Reply:
If possible we should continue to use the reconciler. IMO there are a couple of options for how we can achieve the same results:
- Add an annotation to the node to indicate when the drain attempt begin. Continue to reconcile until an upper time limit has been hit.
- Do the same thing but with an in-memory map from node to timestamp.
Either option would work, the advantage to an annotation is that we would persist that information through a controller restart, though I'm not sure how valuable that really is here.